src/Nfield.Quota/QuotaLevelDefinition.cs

```diff
@@ -2,9 +2,35 @@
 
 namespace Nfield.Quota
 {
-    public class QuotaLevelDefinition
+    public class QuotaLevelDefinition : IEquatable<QuotaLevelDefinition>
     {
         public Guid Id { get; set; }
         public string Name { get; set; }
+
+        public static bool operator ==(QuotaLevelDefinition existingDefinition, QuotaLevelDefinition newDefinition)
+        {
+            return existingDefinition?.Equals(newDefinition) ?? false;
+        }
+
+        public static bool operator !=(QuotaLevelDefinition existingDefinition, QuotaLevelDefinition newDefinition)
+        {
+            return !(existingDefinition == newDefinition);
+        }
+
+        public override bool Equals(object obj)
+        {
+            var other = obj as QuotaLevelDefinition;
+            return other != null && Equals(other);
+        }
+
+        public override int GetHashCode()
+        {
+            return Id.GetHashCode() ^ Name.GetHashCode();
```

Comment:
I know that if you implement `Equals` you should implement `GetHashCode` but this implementation seems wrong because `Name` and `Id` are mutable. Maybe `base.GetHashCode` is actually better.